### ISSUE TITLE
User Management

### ISSUE THREAD

**Issue by stdavis** [agrc/electrofishing#8]
The application will require a login to edit the application (user login will not allow anything else)

Implement PermissionProxy.

**Comment by jpechmann** [comment]
Leaning towards using UMD. DWR would like to use the same log in for future applications, and would also like to avoid users making a new user name and password.

**Comment by stdavis** [comment]
We probably need to wait for the new REST service coming for UMD. Talk to Kyle Burkhardt for status updates.

**Comment by steveoh** [comment]
It sounds like the standard OpenAM module can be placed in front of this website. I was told there is one role and no public view. I believe DTS has a user management console if there needs to be user approval. I do not think we will be needing permission proxy. If this is all true, let's create a ticket to have openam installed.

**Comment by stdavis** [comment]
Would we just have open AM secure the website and arcgis server services?

**Comment by steveoh** [comment]
Have you done that already? I'm not sure how OpenAM would work with server. 

Since the site is secure you could request a token from the client code.

**Comment by stdavis** [comment]
I haven't done anything with security for this app yet.

You mean we could hard code the ArcGIS Server username and password into the JavaScript? I think that we use OpenAM to secure the vista service on mapserv.

**Comment by steveoh** [comment]
Yeah I was generally referring to securing a service with open am. If you've done it with vista, then it could work here also.

Yes I was saying we could hardcode a user/password with access to the fishing services.

**Comment by stdavis** [comment]
OK. I propose that we attempt to secure both the app and service with OpenAM and leave the service open as far as ArcGIS Server is concerned and see if it works. That seems like the most simple solution.

**Comment by steveoh** [comment]
INC0820299

**Comment by steveoh** [comment]
[openam docs](https://kb.dts.utah.gov/pages/viewpage.action?pageId=97093058)
